soc/nordic/nrf54l/Kconfig.defconfig.nrf54l15_cpuflpr

```diff
@@ -0,0 +1,15 @@
+# Nordic Semiconductor nRF54L15 MCU
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF54L15_CPUFLPR
+
+config RISCV_HAS_CPU_IDLE
+	bool
+
+config NUM_IRQS
+	int
```

Comment:
`int` not needed, this is just defaulting

Reply:
done